### PR TITLE
[Multi-Region] Adding pipeline changes for Multi region load tests

### PR DESCRIPTION
**Pull request by venkatr21**
1. Adding multi region support on the ADO pipelines
2. Bump up the data plane API version

### CODE REVIEW

**Review by prativen — DISMISSED** [review]

**Comment by mohitpavan** [comment]
make sure the pr builds are running fine now, its just authentication issue, just ask radhika to update the spn in the github secret. lets make it a habit to wait for builds to be succesful instead of surpassing.

**Comment by mohitpavan** [comment]
main.js missing.

File: lib/main.js [hunk 1]
```diff
@@ -191,7 +191,7 @@ function uploadTestPlan() {
         let retry = 5;
         let filepath = map.getTestFile();
         let filename = map.getFileName(filepath);
-        var urlSuffix = "tests/" + testId + "/files/" + filename + "?api-version=" + util.apiConstants.tm2023Version;
+        var urlSuffix = "tests/" + testId + "/files/" + filename + "?api-version=" + util.apiConstants.tm;
```
Comment (mohitpavan) [inline lib/main.js:194]: its showing as util.apiconstants.tm is it some mistake ? or the pr is showing up like that ?

**Comment by mohitpavan** [comment]
please add a temporary spn (ask @radhikajain99) to add the same, then the auth will succeed, from next time we should not need this anyway, we can go with WIF

**Review by mohitpavan — DISMISSED** [review]

**Review by radhikajain99 — APPROVED** [review]

**Pull request merged**